src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -1941,24 +1947,28 @@ public int HeightSpecification
             set
             {
                 _heightPolicy = value;
-                if (_heightPolicy >= 0)
+                if( _oldHeightPolicy != _heightPolicy )
```

Comment:
The policy is under 0 means that user set LayoutParamPolicies.

In this case, size should not be set.

So, Please check if policy is under 0 too.
( _oldHeightPolicy !=  _heightPolicy && _heightPolicy >= 0)

Reply:
Never mind. I checked again and found that it was on your code already